dom/fuzzer/modules/style-sheets.js

```diff
@@ -434,9 +462,11 @@ var fuzzerRandomClasses = (function() {
     return Random.index(CSSPropList);
   }
 
-  function randomDeclaration()
+  function randomDeclaration(prop)
   {
-    var prop = randomProperty();
+    if (prop === undefined) {
+      prop = randomProperty();
+    }
```

Comment:
This doesn't seem right to me.

* It should be a separate commit, or part of the commit that uses it.
* Your other module should probably use propertyValue(prop) instead of randomDeclaration(prop). Maybe some of the munging that happens here should move into propertyValue. And the 'modifyNumbersInString' call could be moved all the way into importPropertyDatabase, since it doesn't need to apply to values that were generated by the fuzzer.